lit_nlp/client/services/selection_service.ts

```diff
@@ -25,9 +25,12 @@ import {SelectionObservedByUrlService} from './url_service';
 
 /**
  * The AppState interface for working with the SelectionService
+ * TODO(lit-dev): not sure why this is here, other than possibly limiting
+ * circular dependencies.
```

Comment:
correct

Reply:
ah sorry, this was my own note from developing earlier, will remove :)